backend/src/data/sessionsDataAccess.ts

```diff
@@ -0,0 +1,53 @@
+import { pool } from '../config/db'
+import { v4 as uuidv4 } from 'uuid';
+import { ISession } from '../types/session';
+import { ISessionAccessCode } from '../types/sessionAccessCode';
+
+export const createSession = async (): Promise<ISession | null> => {
+    try {
+   
+  
+      const query = `
+        INSERT INTO sessions (id)
+        VALUES ($1) RETURNING id, active, match_found, created_at;
+      `;
+  
+      const data = [
+        uuidv4()
+      ];
+  
+      const result = await pool.query(query, data);
+  
+      return result.rows[0] || null;
```

Comment:
Why do we need to return null?

Reply:
This is a practice I use routinely to avoid returning undefined. It's just to help to make it clear what the purpose is.